### ISSUE TITLE
When deeplima is executed on a short file with tokenizer+tagger, it produces no output

### ISSUE THREAD

**Issue by kleag** [aymara/lima#173]
**Describe the bug**
When deeplima is executed on a short file with tokenizer+tagger, it produces no output.

**To Reproduce**
1. Execute
```bash
deeplima --tok-model $HOME/.local/share/lima/resources/RnnTokenizer/ud/tokenizer-eng-UD_English-EWT.pt --tag-model $HOME/.local/share/lima/resources/RnnTagger/ud/tagger-eng-UD_English-EWT.pt test-eng10.txt
```
with `test-eng10.txt` a short file (like 9 tokens).
2. It finishes without the expected CoNLL-U output

**Expected behavior**
The expected CoNLL-U output should be printed.

**Comment by kleag** [comment]
In fact, there is no output when there is no sentence nor paragraph break at the end of the text.

**Comment by kleag** [comment]
Corrected in commit [c45653d](https://github.com/aymara/lima/commit/c45653d0611af6b4c76f28e51a3756ea0806534f)
